examples/optional_redirect/src/tests.rs

```diff
@@ -0,0 +1,38 @@
+use super::rocket;
+use rocket::testing::MockRequest;
+use rocket::http::{Method, Status};
+
+fn test_200(uri: &str, expected_status: Status, expected_body: String) {
```

Comment:
This function is called `test_200`, so you shouldn't need the `expected_status` parameter. Test for Status::Ok directly below instead.